sympy/physics/control/control_plots.py

```diff
@@ -801,6 +820,8 @@ def bode_phase_numerical_data(system, initial_exp=-5, final_exp=5, **kwargs):
         The only variable in the transfer function should be
         the variable of the Laplace transform.
 
+        When incorrect frequency or phase units are given as input.
```

Comment:
Maybe remove the empty line above? (The other case is more likely to cause the error, but maybe this one as well.)